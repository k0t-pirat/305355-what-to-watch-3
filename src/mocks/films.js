const featuredFilm = {
  title: `My Film`,
  genre: `comedy`,
  realeaseDate: 1988
};

const films = [
  {
    title: `Какие-то там твари`,
    image: `fantastic-beasts-the-crimes-of-grindelwald.jpg`,
  },
  {
    title: `мукбуС`,
    image: `macbeth.jpg`,
  },
  {
    title: `Расподия или что-то ещё`,
    image: `bohemian-rhapsody.jpg`,
  },
  {
    title: `Строитель великой самолётной империи`,
    image: `aviator.jpg`,
  },
  {
    title: `Нужно поболтать`,
    image: `we-need-to-talk-about-kevin.jpg`,
  },
  {
    title: `Что мы здесь делаем`,
    image: `what-we-do-in-the-shadows.jpg`,
  },
  {
    title: `Ревенунт`,
    image: `revenant.jpg`,
  },
  {
    title: `Английский Йонни`,
    image: `johnny-english.jpg`,
  },
];

export {featuredFilm, films};
