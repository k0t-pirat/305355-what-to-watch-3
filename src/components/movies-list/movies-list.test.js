import React from 'react';
import renderer from 'react-test-renderer';
import {films} from '../../mocks/films.js';
import MoviesList from './movies-list.js';

it(`MoviesList renders correctly`, () => {
  const tree = renderer.create(
      <MoviesList
        films = {films}
      />
  ).toJSON();

  expect(tree).toMatchSnapshot();
});
