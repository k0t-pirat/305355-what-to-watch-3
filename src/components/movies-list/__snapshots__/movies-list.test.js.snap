// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MoviesList renders correctly 1`] = `
<div
  className="catalog__movies-list"
>
  <article
    className="small-movie-card catalog__movies-card"
    onMouseOver={[Function]}
  >
    <div
      className="small-movie-card__image"
    >
      <img
        alt="Fantastic Beasts: The Crimes of Grindelwald"
        height="175"
        src="img/fantastic-beasts-the-crimes-of-grindelwald.jpg"
        width="280"
      />
    </div>
    <h3
      className="small-movie-card__title"
    >
      <a
        className="small-movie-card__link"
        href="movie-page.html"
      >
        Какие-то там твари
      </a>
    </h3>
  </article>
  <article
    className="small-movie-card catalog__movies-card"
    onMouseOver={[Function]}
  >
    <div
      className="small-movie-card__image"
    >
      <img
        alt="Fantastic Beasts: The Crimes of Grindelwald"
        height="175"
        src="img/macbeth.jpg"
        width="280"
      />
    </div>
    <h3
      className="small-movie-card__title"
    >
      <a
        className="small-movie-card__link"
        href="movie-page.html"
      >
        мукбуС
      </a>
    </h3>
  </article>
  <article
    className="small-movie-card catalog__movies-card"
    onMouseOver={[Function]}
  >
    <div
      className="small-movie-card__image"
    >
      <img
        alt="Fantastic Beasts: The Crimes of Grindelwald"
        height="175"
        src="img/bohemian-rhapsody.jpg"
        width="280"
      />
    </div>
    <h3
      className="small-movie-card__title"
    >
      <a
        className="small-movie-card__link"
        href="movie-page.html"
      >
        Расподия или что-то ещё
      </a>
    </h3>
  </article>
  <article
    className="small-movie-card catalog__movies-card"
    onMouseOver={[Function]}
  >
    <div
      className="small-movie-card__image"
    >
      <img
        alt="Fantastic Beasts: The Crimes of Grindelwald"
        height="175"
        src="img/aviator.jpg"
        width="280"
      />
    </div>
    <h3
      className="small-movie-card__title"
    >
      <a
        className="small-movie-card__link"
        href="movie-page.html"
      >
        Строитель великой самолётной империи
      </a>
    </h3>
  </article>
  <article
    className="small-movie-card catalog__movies-card"
    onMouseOver={[Function]}
  >
    <div
      className="small-movie-card__image"
    >
      <img
        alt="Fantastic Beasts: The Crimes of Grindelwald"
        height="175"
        src="img/we-need-to-talk-about-kevin.jpg"
        width="280"
      />
    </div>
    <h3
      className="small-movie-card__title"
    >
      <a
        className="small-movie-card__link"
        href="movie-page.html"
      >
        Нужно поболтать
      </a>
    </h3>
  </article>
  <article
    className="small-movie-card catalog__movies-card"
    onMouseOver={[Function]}
  >
    <div
      className="small-movie-card__image"
    >
      <img
        alt="Fantastic Beasts: The Crimes of Grindelwald"
        height="175"
        src="img/what-we-do-in-the-shadows.jpg"
        width="280"
      />
    </div>
    <h3
      className="small-movie-card__title"
    >
      <a
        className="small-movie-card__link"
        href="movie-page.html"
      >
        Что мы здесь делаем
      </a>
    </h3>
  </article>
  <article
    className="small-movie-card catalog__movies-card"
    onMouseOver={[Function]}
  >
    <div
      className="small-movie-card__image"
    >
      <img
        alt="Fantastic Beasts: The Crimes of Grindelwald"
        height="175"
        src="img/revenant.jpg"
        width="280"
      />
    </div>
    <h3
      className="small-movie-card__title"
    >
      <a
        className="small-movie-card__link"
        href="movie-page.html"
      >
        Ревенунт
      </a>
    </h3>
  </article>
  <article
    className="small-movie-card catalog__movies-card"
    onMouseOver={[Function]}
  >
    <div
      className="small-movie-card__image"
    >
      <img
        alt="Fantastic Beasts: The Crimes of Grindelwald"
        height="175"
        src="img/johnny-english.jpg"
        width="280"
      />
    </div>
    <h3
      className="small-movie-card__title"
    >
      <a
        className="small-movie-card__link"
        href="movie-page.html"
      >
        Английский Йонни
      </a>
    </h3>
  </article>
</div>
`;
